warehouse/macros/generate_schema_name.sql

```diff
@@ -0,0 +1,13 @@
+{% macro generate_schema_name(custom_schema_name, node) -%}
```

Comment:
optional: Maybe some explanation (comment) would be useful here to explain what is going on

Reply:
yes I will link to the dbt docs